#!/usr/bin/env python
PACKAGE = "dynamic_tutorials"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("field_x",    double_t,    0, "Parameter for controlling the field in x direction", 0.001,  0, 1)
gen.add("field_y",    double_t,    0, "Parameter for controlling the field in y direction", 0.001,  0, 1)
gen.add("field_z",    double_t,    0, "Parameter for controlling the field in z direction", 0.001,  0, 1)


exit(gen.generate(PACKAGE, "dynamic_tutorials", "Tutorials"))